sprocs/update_group_pogil_role.sql

```diff
@@ -0,0 +1,19 @@
+DROP FUNCTION IF EXISTS update_group_pogil_role(enum_pogil_role,bigint,bigint);
+CREATE OR REPLACE FUNCTION
+    update_group_pogil_role (
+        arg_group_role enum_pogil_role, -- This is the new POGIL role!
+        arg_user_id bigint,
+        arg_group_id bigint
```

Comment:
Put the user_id and group_id as the first args, because they are the primary key of the row. Also put the group_id first because that's the primary key order. Also don't use `arg_group_role` when the column is called `pogil_role` -- instead use `arg_pogil_role`. So the args will be `arg_group_id, arg_user_id, arg_pogil_role`.